core/src/main/java/hudson/model/Fingerprint.java

```diff
@@ -1234,74 +1239,34 @@ public int compare(FingerprintFacet o1, FingerprintFacet o2) {
     }
 
     /**
-     * Save the settings to a file.
+     * Save the Fingerprint in the Fingerprint Storage
      * @throws IOException Save error
      */
     public synchronized void save() throws IOException {
-        if(BulkChange.contains(this))   return;
+        if(BulkChange.contains(this)) {
+            return;
+        }
 
-        long start=0;
+        long start = 0;
```

Comment:
BTW please avoid gratuitous reformatting of otherwise untouched code.

Reply:
@jglick Thank you so much for reviewing! https://github.com/jenkinsci/jenkins/pull/4731#discussion_r436557928
I changed it because of feedback, Ill change it back, no worries :)

Reply:
To be clear, the new format is closer to typical styleguides, I just advise keeping PR diffs as small as practical; if there is a desire to clean up formatting of code, better to do that in dedicated PRs.